modules/pushapk_scriptworker/manifests/init.pp

```diff
@@ -112,6 +112,8 @@
                     content => $google_play_accounts['fenix-nightly']['certificate'];
                 "${root}/fenix_beta.p12":
                     content => $google_play_accounts['fenix-beta']['certificate'];
+                "${root}/fenix_production.p12":
```

Comment:
Should we name this `fenix_release` to be consistent? I see `focus` has something similar `focus_release`...

Reply:
Good question: `focus` uses a different deployment scheme - though it has multiple channels (nightly/production), they're all represented under the same Google Play Store entry. So, they all have the same package name, and are signed with the same key.

So, for terminology: if an app is made available for real customers to use (e.g.: the channels nightly/beta/production), then the app is "released". This makes builds for these channels "release builds". Since Focus uses the same signing/push credentials for all of its channels, then we can name the credentials accordingly: `focus_release`

Fenix is different, because it has a different app for every channel. So, we need separate credentials for our release channels. I have named each credential to match the channel name (nightly/beta/production)